transform/models/marts/dim_repos.sql

```diff
@@ -0,0 +1,3 @@
+Select distinct repo_id,
```

Comment:
Style: Let's try to respect casing (you can go with either upper or lowercase for SQL, but once chosen better to stick to one or the other).